lib/charms/operator_libs_linux/v0/apt.py

```diff
@@ -1191,8 +1206,10 @@ def _get_key_by_keyid(keyid: str) -> str:
             "https://keyserver.ubuntu.com" "/pks/lookup?op=get&options=mr&exact=on&search=0x{}"
         )
         curl_cmd = ["curl", keyserver_url.format(keyid)]
-        # use proxy server settings in order to retrieve the key
-        return check_output(curl_cmd).decode()
+        with tracer.start_as_current_span(curl_cmd[0]) as span:
+            span.set_attribute("argv", curl_cmd)
```

Comment:
I'm not sure if we would want to have the key ID in the span attributes. It's not secret, but it does feel personal, so not a great fit.

Reply:
You have a point.
Then again, this is a network request with a URL parameter.
It's pretty much bound to be logged on the server.

Note that the key id is the id of key being imported, that is some PPA being added (most likely, the library doesn't enforce that).
It's not a personal key.